packages/arm-core/src/components/Marker.tsx

```diff
@@ -12,9 +12,10 @@ type Props = {
   latLng: LatLngExpression
   events?: { [key: string]: LeafletEventHandlerFn }
   options?: MarkerOptions
+  setInstance?: (markerInstance: MarkerType | undefined) => void
```

Comment:
Instead of `| undefined` you can also make this argument optional.

```suggestion
  setInstance?: (markerInstance?: MarkerType) => void
```